js/oop/Factorial.js

```diff
@@ -21,7 +38,10 @@ document.getElementById("form").addEventListener("submit", function (event) {
     const n = event.target["n"].value;
     const method = event.target["method"].value;
 
-    const result = new Factorial(n);
+    const result =
+      method === "loop"
+        ? new Factorial(n).loop()
+        : new Factorial(n).recursive();
```

Comment:
This is not how OOP was created on purpose. Could be changed much better than this.

Reply:
Change it, but not sure if it correctly

Reply:
Create a new method count() at the Factorial class. Then move all of the incoming "method" checking from this main code into that count() method I mentioned before. So now you just do a simple method call count() at the main code.